process/consensus.md

```diff
@@ -87,20 +90,22 @@ We differentiate the following cases:
    three contributors of different affiliations is acceptable as long as there
    is no objection. Consensus will be deemed to not have been reached if
    interested parties did not sign off. At any point in time a contributor can
-   request that final consensus be delayed to an in-person meeting. In this
+   request that final consensus be delayed to a subgroup, or a CG meeting. In this
    case, the chair puts the item on the group's agenda of upcoming
    discussions.
 3. Substantial technical changes or additions are usually carried in their own
    GitHub repository by a champion. It is critical that these proposals be able
    to evolve quickly without much process. Early on in such a proposal's
    lifetime no consensus is needed and a single champion can modify the proposal
    at will. As the proposal matures it is expected that the champion will seek
-   collaborators to form a small group. Gauging consensus in the small group is
-   left up to the champion, with input from the chair. When a proposal is near
-   maturity the champion shall bring it to a meeting and seek wider
+   collaborators to form a subgroup. Gauging consensus in the small group is
+   left up to the champion, gauging consensus in a subgroup is left to the chair,
```

Comment:
```suggestion
   left up to the champion, and gauging consensus in a subgroup is left to the chair
```

Reply:
Since "small groups" include subgroups, it is ambiguous whether the chair or the champion declares consensus in cases where the small group is also a subgroup. It would be good to clarify that "if the small group is a subgroup, then the chair or co-chairs of that subgroup gauge consensus, and otherwise the champion guages consensus."

Reply:
I hit resolve by accident but, small group consensus especially online (ex: on PRs, issues), is usually up to the champion, and sometimes up to the subgroup chair. Whereas consensus at Subgroup meetings is usually upto the Subgroup chait. Will walk back the edit, and reword this better.

Reply:
I've reworked this paragraph a little bit to include subgroups, ptal.